src/pages/Auth/Register/Register.js

```diff
@@ -29,8 +29,12 @@ export default function Register(props) {
     e.preventDefault();
     setLoading(true);
 
-    const res = await axios.post('users.json');
- 
+    const res = await axios.post('https://identitytoolkit.googleapis.com/v1/accounts:signUp?key=[AIzaSyDTk6knBjnCpxswc9N1LL4YnfGHp_194yA]', {
+      email: form.email.value,
+      password: form.password.value,
+      returnSecureToken: true
+    });
+    console.log(res.data)

```

Comment:
Create folder "services" and create file auth.js. In this file create "register" method with this code.
The file can contain also "login" method and "currentUser" (for fetching user details)